src/SBMLBioModelsRepository.jl

```diff
@@ -82,16 +82,17 @@ the total size of the SBML is like 5 MB
 there's got to be a better way to access the test-suite
 did anyone even intend for people to use this?
 """
-function sbml_test_suite(dir="data/sbml_test_suite_models/")
-    !isdir(dir) && mkdir(dir)
-    repo_p = "data/sbml-test-suite/"
-    run(`git clone "https://github.com/sbmlteam/sbml-test-suite/" $(repo_p)`)
-    p = "data/sbmlcases/semantic/"
-    ds = filter(isdir, readdir(p; join=true))
+function sbml_test_suite(dir=joinpath("data","sbml_test_suite_models"))
+    repo_p = joinpath("data","sbml-test-suite")
+    !isdir(dir) && begin mkpath(dir)   
+        run(`git clone https://github.com/sbmlteam/sbml-test-suite.git $(repo_p)`)
+    end
+    p = joinpath("cases","semantic")
+    ds = filter(isdir, readdir(joinpath(repo_p,p); join=true))
     fns = reduce(vcat, glob.("*.xml", ds))
     fs = map(x -> splitdir(x)[end], fns)
     dsts = normpath.(dir .* fs)
-    cp.(fns, dsts)
+    cp.(fns, dsts, force=true)
```

Comment:
I don't think you should need this, what most likely happened is you ran it once, it instantiated some files at dsts, then failed. So when you tried to rerun it, it needed a `force`

Reply:
Hmm. Don't you think function behaviour should be consistent between multiple calls? Just in case sb calls it twice?

Reply:
in case user provides dir that has other stuff in it, you don't wanna silently delete

Reply:
cp(;force) is a sharp knife